src/main/scala/is/hail/utils/package.scala

```diff
@@ -598,6 +598,26 @@ package object utils extends Logging
       case _: MatchError => None
     }
   }
+
+  // unlike Scala's String.split, the length of the result is one more
+  // than the nubmer of sep characters in s
```

Comment:
typo here "nubmer"

Reply:
I think this is just `str.split(sep, -1)`. It's probably faster though (though not by much, since String.split does optimize the 1-char case).

Reply:
Ah,  negative returns trailing empties, thanks.